pipeline/pipelines/pipeline_factory.py

```diff
@@ -22,9 +22,9 @@
 
 
 def make_pipeline_flow(pipeline_name, **kwargs):
-    """This factory function to make veneer pipeline.
+    """Factory function to make a GAPIC pipeline.
 
-    Because veneer pipeline is using OpenStack Taskflow, this factory function
+    Because GAPIC pipeline is using OpenStack Taskflow, this factory function
```

Comment:
Good catch. Annoying to fix since I had to reformat all the lines (pep8 strictly enforces an 80 character limit per line).